aws/database/opensearch/client/elasticsearch_client.py

```diff
@@ -1,6 +1,8 @@
 import elasticsearch
 from elasticsearch import Elasticsearch
 
+import urllib3
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
Seems potentially problematic without further context.

Perhaps, a comment explaining why this warning is being disabled could be added?

Reply:
Good point, it does look "dodgy". It's because we're connecting to Localhost and it keeps complaining about the lack of SSL.

Added a comment: ce0dc70e5947313c6efa1fa32dcc01b802cb6f2f